### PR TITLE
[INLONG-9470][Manager] Fix the problem of failed to verify if the namespace exists

### LINKED ISSUE #9470: [Bug][Manager] Failed to verify if the namespace exists
### What happened

Failed to verify if the namespace exists.

### What you expected to happen

Failed to verify if the namespace exists.

### How to reproduce

Create an existing namespace.

### Environment

_No response_

### InLong version

master

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by fuweng11**
### Prepare a Pull Request

- Fixes #9470

### Motivation

Fix the problem of failed to verify if the namespace exists.

### Modifications

Fix the problem of failed to verify if the namespace exists.

### CODE REVIEW

File: inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/util/HttpUtils.java [hunk 1]
```diff
@@ -95,6 +95,36 @@ public static <T> T request(RestTemplate restTemplate, String url, HttpMethod me
         }
     }
 
+    /**
+     * Send an HTTP request by the given rest template.
+     */
+    public static <T> T request(RestTemplate restTemplate, String[] urls, HttpMethod method,
+            String param, HttpHeaders header, Class<T> cls) throws Exception {
+        ResponseEntity<String> exchange;
+        for (int i = 0; i < urls.length; i++) {
```
Comment (vernedeng) [inline inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/util/HttpUtils.java:104]: NPE

File: inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/cluster/pulsar/PulsarClusterInfo.java [hunk 2]
```diff
@@ -55,4 +58,13 @@ public PulsarClusterRequest genRequest() {
         return CommonBeanUtils.copyProperties(this, PulsarClusterRequest::new);
     }
 
+    public String[] getAdminUrls(String urlSuffix) {
+        String adminUrl = this.getAdminUrl();
```
Comment (vernedeng) [inline inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/cluster/pulsar/PulsarClusterInfo.java:62]: NPE

Comment (fuweng11) [reply inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/cluster/pulsar/PulsarClusterInfo.java:62]: Done.

File: inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/util/HttpUtils.java (hunk 1 above)
Comment (fuweng11) [reply inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/util/HttpUtils.java:104]: Done.

**Review by vernedeng — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Comment by dockerzhang** [comment]
@haibo-duan PTAL, thanks.

**Review by haibo-duan — APPROVED** [review]

**Pull request merged**